style.css

```diff
@@ -0,0 +1,210 @@
+/* the + sign is used to select the sibling that comes immediately next to the element you selected.
+in this case it is the div with the answer class next to the div with the active class.
+this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+
+
+/* Hero video styling*/
+#hero {
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#hero::after {
+  width: 100%;
+  height: 100%;
+  content: '';
+  position: absolute;
+  z-index: 10;
+  top: 0;
+  top: 0;
+  background: rgba(0, 0, 0, 0.5)
+}
+
+#hero video {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 5;
+  object-fit: cover;
+}
+
+#hero .content {
+  position: relative;
+  z-index: 20;
+  text-align: center;
+  color: white;
+}
+
+#hero .content h1 {
+  font-size: 1.5em;
+  Font-Family: 'Raleway', Sans-Serif;
+  margin-bottom: 0.5rem;
+
+}
+
+#hero .content h2 {
+  font-weight: 400;
+  font-size: 1rem;
+  margin: 0;
+  Font-Family: 'Open Sans', Sans-Serif;
+
+}
+
+#hero .content p {
+  font-size: 1rem;
+}
+
+#signbutton:hover {
+  cursor: wait;
+}
+/* form */
+.form {
+  position: relative;
+  z-index: 18;
+  display: inline-block;
+  text-align: left;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 12px 20px;
+
+}
+
+input[type=text],
+input[type=email],
+input[type=text],
+input[type=password],
+select,
+textarea {
+  width: 270px;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+  font-size: 10px;
+}
```

Comment:
I like the explicitness here with 
```css
input[type=*]
```  
By doing this, you eliminate problems that could occur with the order of the CSS declarations and the style for 
```css
input[type=submit]
``` 
being overwritten by something like
```css
input, select, textarea {
}
```